### PR TITLE
task/27174-AWM -WebApp-User-create-layout

### PR DESCRIPTION
**Pull request by hungdmh**
https://edu-redmine.sun-asterisk.vn/issues/27102

image

### CODE REVIEW

**Review by ducminh99 — APPROVED** [review]

**Comment by hungdmh** [comment]
ready

**Comment by hungdmh** [comment]
ready

File: awm-webapp/styles/antd.less [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@import '~antd/dist/antd.less';
+
+@primary-color: #1890ff; // primary color for all components
```
Comment (hieplt-0914) [inline awm-webapp/styles/antd.less]: bỏ comment đi em

**Comment by hungdmh** [comment]
ready

**Pull request merged**